### DISCUSSION TITLE
Call for the Translation of the User Manual and Interface of the Semi-Automatic Classification Plugin

### DISCUSSION THREAD

**Discussion by semiautomaticgit** [semiautomaticgit/SemiAutomaticClassificationPlugin#309] [General]
The new version 8 of the Semi-Automatic Classification Plugin (SCP) has been recently released, along with the updated user manual.
I kindly invite you to translate the user manual and the program interface to your language. Your contribution is fundamental for every user who doesn't speak English. It also can be a fun way to deeply learn about the SCP.

https://fromgistors.blogspot.com/2023/10/call-for-translation.html

**Comment by rnanclares** [comment]
Hi Luca, I just joined the team. I could do Spanish (Spain) and Spanish (Mexico) but I don't know if that's possible through transifex. I might just try the .po files.

**Reply by semiautomaticgit** [reply]
Hi @rnanclares , thank you very much for your availability!

> I could do Spanish (Spain) and Spanish (Mexico) but I don't know if that's possible through transifex. I might just try the .po files.

Spanish (Spain) and Spanish (Mexico) are available through Transifex, but of course .po files are fine if it is better for you.

**Comment by rnanclares** [comment]
![image](https://github.com/semiautomaticgit/SemiAutomaticClassificationPlugin/assets/11431207/64ee6e2a-1afd-4dd0-ac24-98c1bad6122c)
Sorry to ask here but I don't know where is the right place to ask. Do we need to translate those sort strings? Or are they part of the code?

**Reply by semiautomaticgit** [reply]
Thank you for asking.
No, those strings with underscore shouldn't be translated because those are reference to image icons.
It is actually strange those are listed.

Thank you very much for your effort in translating!

**Comment by rnanclares** [comment]
Got it. Thanks

**Reply by semiautomaticgit** [reply]
Thank you very much @rnanclares !
You can already read the translated documentation at https://semiautomaticclassificationmanual.readthedocs.io/es/latest/main_interface_window.html
